examples/DQMC/from_fcidump/h2o/h2o.py

```diff
@@ -0,0 +1,58 @@
+import sys, os
+import numpy
+import numpy as np
+import QMCUtils
+
+# these need to be provided
+dice_binary = "/projects/anma2640/newDice/Dice/Dice"
+vmc_root = "/projects/anma2640/VMC/master/VMC"
+
+
+norb_frozen = 1
+nelec = 10
+norb_act = 6 
+nelec_act = 8
+norb_core = (nelec-nelec_act)//2 -norb_frozen # mc.ncore  
+chol_cut = 1e-5
+
+spin = 0 #alpha-beta                                   
+nproc = 1                                              
```

Comment:
maybe best not to use nproc (definitely not just 1), just let mpirun use all available processes